<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ListenSub</class>
 <widget class="QWidget" name="ListenSub">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>666</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>666</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>666</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>视唱练耳小助手</string>
  </property>
  <widget class="QPushButton" name="BackListen">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="download">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>530</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #808F9F;
    border: none;
    border-radius: 10px;
    icon: url(:/MyPicture/dl.png);
    icon-size: 35px 35px;
}
QPushButton:hover {
    background-color: #6B7A8A;
}
QPushButton:pressed {
    background-color: #5A6877;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="play">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>380</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #FBF1E3;
    border: 6px solid #433F67;
    border-radius: 50px;
    color: white;
    padding-left: 10px;
    padding-top: 4px;
    icon: url(:/MyPicture/p.png);
}
QPushButton:hover {
    background-color: #F5E7D3;
}
QPushButton:pressed {
    background-color: #EFDDC3;
}
QPushButton:checked {
    icon: url(:/MyPicture/stp.png);
    padding-left: 0px;
    padding-top: 4px;
    icon-size: 20px 20px;
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="refresh">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>20</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正小标宋简体</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 30, 129);
</string>
   </property>
   <property name="text">
    <string>下一个</string>
   </property>
  </widget>
  <widget class="QPushButton" name="test">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>530</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正小标宋简体</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #808F9F;
    border: none;
    border-radius: 10px;
    icon: url(:/MyPicture/tst.png);
    icon-size: 45px 45px;
    text-align: center;
    padding-left: 0px;
    color: white;
}
QPushButton:hover {
    background-color: #6B7A8A;
}
QPushButton:pressed {
    background-color: #5A6877;
}</string>
   </property>
   <property name="text">
    <string>自测</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>185</x>
     <y>110</y>
     <width>840</width>
     <height>350</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QScrollArea {
    border: none;
    background-color: transparent;
    outline: none;
}
QScrollArea &gt; QWidget &gt; QWidget {
    background-color: transparent;
    border: none;
    outline: none;
}
QScrollBar:vertical {
    border: none;
    background: transparent;
    width: 10px;
    margin: 0px;
}
QScrollBar::handle:vertical {
    background: #433F67;
    min-height: 20px;
    border-radius: 5px;
}
QScrollBar::add-line:vertical, QScrollBar::sub-line:vertical {
    height: 0px;
}
QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
    background: none;
}
QScrollArea &gt; QWidget {
    border: none;
    outline: none;
}</string>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>840</width>
      <height>350</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="yuepuLabel">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>新宋体</family>
         <pointsize>18</pointsize>
         <bold>false</bold>
         <hintingpreference>PreferNoHinting</hintingpreference>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::Shape::NoFrame</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <zorder>BackListen</zorder>
  <zorder>download</zorder>
  <zorder>refresh</zorder>
  <zorder>test</zorder>
  <zorder>scrollArea</zorder>
  <zorder>play</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
